lib/TPP/TileConsumerAndFuseProducers.cpp

```diff
@@ -533,7 +529,20 @@ fuseWithEltwise(RewriterBase &rewriter, TilingInterface consumer,
   return tileAndFuseResult;
 }
 
-static SmallVector<int64_t> getDefaultTileSizes() { return {32, 32}; }
+static SmallVector<int64_t> getDefaultTileSizes(linalg::LinalgOp linalgOp) {
+  // Blocked convolutions are tiled and fused along the three outermost parallel
+  // loops to expose a BRGEMM.
+  if (linalgx::utils::isBlockedConvolution(linalgOp))
+    return {1, 1, 1};
+  // Matmuls are tiled and fused along i and j with 32.
+  if (tpp::utils::isTppMatmul(linalgOp))
+    return {32, 32};
```

Comment:
`pack-matmul` takes 3 values for its block-factors

Reply:
Yes, but for tile and fuse only two are necessary

Reply:
Oh yes, sorry. That's completely unrelated - all good.